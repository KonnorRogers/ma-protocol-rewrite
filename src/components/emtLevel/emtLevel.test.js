import React from "react"
import { render } from "@testing-library/react"
import EMTLevel from "./emtLevel.js"

test("Renders without error", () => {
  const { asFragment } = render(
    <EMTLevel level="paramedic">
      <ul>
        <li>Protocols</li>
      </ul>
    </EMTLevel>
  )

  expect(asFragment).toMatchSnapshot()
})

test("Renders med control without error", () => {
  const { asFragment } = render(
    <EMTLevel level="advanced" medControl={true}>
      <ul>
        <li>More Protocols</li>
      </ul>
    </EMTLevel>
  )

  expect(asFragment).toMatchSnapshot()
})
