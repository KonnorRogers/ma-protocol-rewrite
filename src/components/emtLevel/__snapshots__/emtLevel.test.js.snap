// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders med control without error 1`] = `
<DocumentFragment>
  .emotion-1 {
  margin: 0;
  font-size: 1.2rem;
  color: #ffbc0d;
}

@media only screen and (min-width:640px) {
  .emotion-1 {
    font-size: 1.3rem;
  }
}

.emotion-0 {
  color: #ffbc0d;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-0:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-2 {
  overflow-x: scroll;
  margin: 0.5rem auto 1rem auto;
  border: 2px solid #ffbc0d;
  border-radius: 12px;
  font-size: 1.15rem;
  padding: 0.75rem 0.45rem 0 0.45rem;
}

.emotion-2 > .emt-box {
  padding: 1.5rem;
}

.emotion-2 ul {
  padding-left: 1rem;
  padding-right: 0.5rem;
}

@media only screen and (min-width:640px) {
  .emotion-2 {
    font-size: 1.25rem;
    padding: 0.75rem 1.5rem 0 1.5rem;
  }
}

@media only screen and (min-width:400px) {
  .emotion-2 {
    overflow: auto;
  }
}

.emotion-1 {
  margin: 0;
  font-size: 1.2rem;
  color: #ffbc0d;
}

@media only screen and (min-width:640px) {
  .emotion-1 {
    font-size: 1.3rem;
  }
}

.emotion-0 {
  color: #ffbc0d;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-0:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-2 {
  overflow-x: scroll;
  margin: 0.5rem auto 1rem auto;
  border: 2px solid #ffbc0d;
  border-radius: 12px;
  font-size: 1.15rem;
  padding: 0.75rem 0.45rem 0 0.45rem;
}

.emotion-2 > .emt-box {
  padding: 1.5rem;
}

.emotion-2 ul {
  padding-left: 1rem;
  padding-right: 0.5rem;
}

@media only screen and (min-width:640px) {
  .emotion-2 {
    font-size: 1.25rem;
    padding: 0.75rem 1.5rem 0 1.5rem;
  }
}

@media only screen and (min-width:400px) {
  .emotion-2 {
    overflow: auto;
  }
}

<div
    style="margin: auto; max-width: 800px;"
  >
    <h2
      class="emotion-1"
      id="advanced-med-control"
    >
      <a
        class="emotion-0"
        href="#advanced-med-control"
      >
        EMT-Advanced (Medical Control)
      </a>
    </h2>
    <section
      class="advanced emt-box emotion-2"
    >
      <ul>
        <li>
          More Protocols
        </li>
      </ul>
    </section>
  </div>
</DocumentFragment>
`;

exports[`Renders without error 1`] = `
<DocumentFragment>
  .emotion-1 {
  margin: 0;
  font-size: 1.2rem;
  color: red;
}

@media only screen and (min-width:640px) {
  .emotion-1 {
    font-size: 1.3rem;
  }
}

.emotion-0 {
  color: red;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-0:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-2 {
  overflow-x: scroll;
  margin: 0.5rem auto 1rem auto;
  border: 2px solid red;
  border-radius: 12px;
  font-size: 1.15rem;
  padding: 0.75rem 0.45rem 0 0.45rem;
}

.emotion-2 > .emt-box {
  padding: 1.5rem;
}

.emotion-2 ul {
  padding-left: 1rem;
  padding-right: 0.5rem;
}

@media only screen and (min-width:640px) {
  .emotion-2 {
    font-size: 1.25rem;
    padding: 0.75rem 1.5rem 0 1.5rem;
  }
}

@media only screen and (min-width:400px) {
  .emotion-2 {
    overflow: auto;
  }
}

.emotion-1 {
  margin: 0;
  font-size: 1.2rem;
  color: red;
}

@media only screen and (min-width:640px) {
  .emotion-1 {
    font-size: 1.3rem;
  }
}

.emotion-0 {
  color: red;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-0:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-2 {
  overflow-x: scroll;
  margin: 0.5rem auto 1rem auto;
  border: 2px solid red;
  border-radius: 12px;
  font-size: 1.15rem;
  padding: 0.75rem 0.45rem 0 0.45rem;
}

.emotion-2 > .emt-box {
  padding: 1.5rem;
}

.emotion-2 ul {
  padding-left: 1rem;
  padding-right: 0.5rem;
}

@media only screen and (min-width:640px) {
  .emotion-2 {
    font-size: 1.25rem;
    padding: 0.75rem 1.5rem 0 1.5rem;
  }
}

@media only screen and (min-width:400px) {
  .emotion-2 {
    overflow: auto;
  }
}

<div
    style="margin: auto; max-width: 800px;"
  >
    <h2
      class="emotion-1"
      id="paramedic"
    >
      <a
        class="emotion-0"
        href="#paramedic"
      >
        EMT-Paramedic (Standing Orders)
      </a>
    </h2>
    <section
      class="paramedic emt-box emotion-2"
    >
      <ul>
        <li>
          Protocols
        </li>
      </ul>
    </section>
  </div>
</DocumentFragment>
`;
